content/userguide/index.md

```diff
@@ -71,13 +71,170 @@ sudo systemctl restart mysterium-node.service
 If your `mysterium.node` service is still refusing to start you can report an issue to our support team
 which will help you figure it out as best as they can. 
 
-## Basic myst CLI guide 
+## Basic: Connect using myst commands
 
-In order to start using our dVPN you will have to get familiar with our
-`myst cli` console application.
+After getting your node up and running you should be almost ready to connect. In this section
+we'll go through the steps required to do that.
 
-Initial setup shouldn't take more than a few minutes and after that you'll only need to execute
-a few commands whenever you want to use mysterium dVPN.
+This section will touch on two commands `myst account` and `myst connection` together with
+their subcommands.
+
+These two commands expose the most basic and user friendly way to use your terminal in order
+to connect to the dVPN. For more advanced user guide please follow the section below
+which explains how to use `myst cli`.
+
+### How to use the commands 
+
+As mentioned above we'll use two commands and their subcommands:
+- `myst account`
+- `myst connection`
+
+In order to read more about them you can execute them with a help flag like this:
+
+```bash
+myst account --help
+myst connection --help
+```
+This also works for any sub-commands.
+
+Each command has an output, if the output does not start with `[ERROR]` or `[WARNING]` consider that command a success.
```

Comment:
It is common to use exit codes to detect if the command succeeded or not. let's mention it here too.

Reply:
We dont really print exit codes. Which I guess we should

Reply:
We should not print it. Users can get it when the command finished.